File: src/main/java/ninja/hudy/infosmog/model/RegionType.java
package ninja.hudy.infosmog.model;

public enum RegionType {
    VOIVODESHIP(Values.VOIVODESHIP),
    COUNTY(Values.COUNTY),
    CITY(Values.CITY);

    private String value;

    RegionType(String value) {
        this.value = value;
    }

    public String toString() {
        return value;
    }

    public static class Values {
        public static final String VOIVODESHIP = "VOIVODESHIP";
        public static final String COUNTY = "COUNTY";
        public static final String CITY = "CITY";
    }
}

File: src/main/kotlin/pl/org/smok/infosmog/model/RegionType.kt
package pl.org.smok.infosmog.model

enum class RegionType private constructor(private val value: String) {
    VOIVODESHIP(Values.VOIVODESHIP),
    COUNTY(Values.COUNTY),
    CITY(Values.CITY);

    override fun toString(): String {
        return value
    }

    object Values {
        const val VOIVODESHIP = "VOIVODESHIP"
        const val COUNTY = "COUNTY"
        const val CITY = "CITY"
    }
}
